### PR TITLE
Footer layout

### PR DESCRIPTION
**Pull request by Aselshine**
Issue #:
[Trello](https://trello.com/invite/b/AklPlVwg/ATTI9fdb094d2c14954c792b9e3930d9949a11CEF7C2/solid-practicum-dominos-team-2)

## PR type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### after:
Аннотация 2023-01-31 214108
Аннотация 2023-01-31 214207
Аннотация 2023-01-31 214255

## Does this introduce a breaking change?

- [ ] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [x] Responsive layout and functionality have been tested
  - [x] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/Components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                <div className={styles.first}>
+                       <div className={styles.title}>
+                        <span>Domino's Pizza</span>
+                        <button id={styles.first} onClick={() => setIsOpenDomino(!isOpenDomino)}>
+                            <img className={`${styles.arrow} ${isOpenDomino ? styles.openDomino : ""}` } src={arrow} alt="arrow" />
+                        </button>
+                        </div>
+                        {isOpenDomino && <div className={styles.firstColumn}>
+                        
+                            <a href="https://dominos.ua/uk/kyiv/Pizza/">Menu</a>
```
Comment (aiskairr) [inline src/Components/Footer/Footer.jsx]: Вместо тега a используйте тег Link react-router-dom

File: src/Components/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                        </div>
+                        {isOpenContacts && <div className={styles.thirdColumn}>
+
+                            <div>
+                                <a href="https://dominos.ua/en/kyiv/">info@dominos.ua</a>
+                            </div>
+                            <div>
+                                <a href="https://dominos.ua/en/kyiv/feedback/">Send complaint</a>
+                            </div>
+                            <div></div>
```
Comment (aiskairr) [inline src/Components/Footer/Footer.jsx]: Этот div обаятельный?

File: src/Components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,748 @@
...
+    padding: 2em 3em 2em;
+    justify-content: space-evenly;
+    width: 100%;
+    height: 350px;
+    background-color: #000;
+    color: #fff;
+
+}
+
+a:focus {
```
Comment (aiskairr) [inline src/Components/Footer/Footer.module.scss]: Используйте вложенные стили

File: src/Components/Footer/Footer.module.scss [hunk 4]
```diff
@@ -0,0 +1,748 @@
...
+.second,
+.third {
+    background-color: #000;
+    font-family: Roboto;
+    font-size: 13px;
+    line-height: 2;
+    margin-top: 1em;
+}
+
+button {
```
Comment (aiskairr) [inline src/Components/Footer/Footer.module.scss:61]: Тут тоже

File: src/Components/Footer/Footer.jsx (hunk 2 above)
Comment (Aselshine) [reply src/Components/Footer/Footer.jsx]: deleted

File: src/Components/Footer/Footer.module.scss (hunk 4 above)
Comment (Aselshine) [reply src/Components/Footer/Footer.module.scss:61]: deleted

File: src/Components/Footer/Footer.jsx (hunk 1 above)
Comment (Aselshine) [reply src/Components/Footer/Footer.jsx]: Why?

File: src/Components/Footer/Footer.module.scss (hunk 3 above)
Comment (Aselshine) [reply src/Components/Footer/Footer.module.scss]: done

**Review by aiskairr — APPROVED** [review]

**Pull request merged**